### DISCUSSION TITLE
Problems installing Security Onion 2.4

### DISCUSSION THREAD

**Discussion by ivanrdrgz** [Security-Onion-Solutions/securityonion#11626] [Unsupported Versions]
Hello everyone,

I am setting up a cybersecurity lab and at first I was using version 2.3, but I have decided to upgrade to the latest version as I have read that it is recommended even if it is a test lab. My problem comes when starting the ISO. 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/102434209/d22c154d-65ab-4d24-b627-5677c5a915bf)

When I select the installation, I get stuck at this point.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/102434209/ac9e8b81-73a9-4183-8909-63a2fe32175c)

My lab is being set up on an ESXi server, and the virtual machine where I install Security Onion has 250gb of storage, 16gb of ram and 4 cores. This problem has never happened to me before with the 2.3 installation on the same virtual machine.

Does anyone know what might be the cause of this problem? 

Thank you very much in advance and best regards.

**Comment by rohan0017** [comment]
Try assigning it 8-10 cores. (I suggest 10 cores)

**Reply by ivanrdrgz** [reply]
Hi! Thank you for your answer. I just try with 8 cores (it is the maximum that ESXi allows, at least in the version I have), more RAM and more storage and the error still there.

**Reply by rohan0017** [reply]
what type of deployment is it(standalone, distributed, heavynode etc)

**Reply by ivanrdrgz** [reply]
None for the moment because the error appears long before I select the type of installation, although I know I will have to select a standalone installation.

**Comment by dougburks** [comment]
Did you verify the ISO image as shown at https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO.md?

**Reply by ivanrdrgz** [reply]
Yes, good sign in key and hash

**Reply by Mav1814** [reply]
hi @ivanrdrgz one question, are you trying to install it in a vm?

**Reply by ivanrdrgz** [reply]
Hi @Mav1814 , thank you for your answer. Yes, I am installing it on a VM hosted on an ESXi server. On this same server I have installed version 2.3 of Security Onion, and although I have not been able to install it due to failures at the end of the configuration (this is why I'm trying with the new version), this premature error did not appear.

**Reply by Mav1814** [reply]
I had this problem in proxmox, i needed to change the CPU to Host in order to boot. I never used ESXI server, so i don't know if is possible to change it .  I don't know if it helps.

**Reply by ivanrdrgz** [reply]
What do you mean about change the CPU to Host?

**Reply by Mav1814** [reply]
here, an example


![](https://www.linuxbabe.com/wp-content/uploads/2020/08/proxmox-vm-cpu-type.png)

**Reply by adamz01h** [reply]
Yes you have to use the host flag on the cpu to get the iso to boot.
